server/views/components/formFields/form-page-date-input/macro.njk

```diff
@@ -5,7 +5,7 @@
       govukDateInput(
         mergeObjects(
           params,
-          { id: params.fieldName, namePrefix: params.fieldName, errorMessage: context.errors[params.fieldName], value: context[params.fieldName] }
+          { id: params.fieldName, name: params.fieldName, errorMessage: context.errors[params.fieldName], value: context[params.fieldName] }
```

Comment:
What's the reason behind this change? It seems the `govukDateInput` does expect `namePrefix` rather than `name`

Reply:
This seems to break functionality, as the resulting inputs simply have `name` attributes set to `day`, `month`, and `year`, which means their values aren't submitted to the expected form data

Reply:
🤦  I could have sworn one of those components mentioned namePrefix which was incorrect. might of been another component. I've reverted these changes now